ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/precompiles/AltBN128MulPrecompiledContract.java

```diff
@@ -30,13 +30,24 @@
       new BigInteger(
           "115792089237316195423570985008687907853269984665640564039457584007913129639935");
 
-  public AltBN128MulPrecompiledContract(final GasCalculator gasCalculator) {
-    super("AltBn128Mul", gasCalculator);
+  private final Gas gasCost;
+
+  private AltBN128MulPrecompiledContract(final GasCalculator gasCalculator, final Gas gasCost) {
+    super("AltBN128Mul", gasCalculator);
+    this.gasCost = gasCost;
+  }
+
+  public static AltBN128MulPrecompiledContract byzantium(final GasCalculator gasCalculator) {
+    return new AltBN128MulPrecompiledContract(gasCalculator, Gas.of(40_000));
+  }
+
+  public static AltBN128MulPrecompiledContract istanbul(final GasCalculator gasCalculator) {
+    return new AltBN128MulPrecompiledContract(gasCalculator, Gas.of(8_000));
```

Comment:
Shouldn't this be `6000` gas ?

Reply:
Yep.  I missed it on review because our schedule was split across 3 files. - https://github.com/PegaSysEng/pantheon/pull/1874